.github/workflows/ci.yml

```diff
@@ -210,7 +210,7 @@ jobs:
           setup: |
             mkdir -p artifacts
             mkdir -p utils
-            cp "$(cs get https://github.com/coursier/coursier/releases/download/v2.0.16/cs-aarch64-pc-linux)" utils/cs
+            cp "$(cs get https://github.com/coursier/coursier/releases/download/v2.1.0-RC5/cs-aarch64-pc-linux)" utils/cs
```

Comment:
@lwronski The url here is wrong. See https://github.com/coursier/coursier/pull/2658, the aarch64 files are hosted here now: https://github.com/VirtusLab/coursier-m1/releases.
However that release page does not have RC5 binaries yet...

Also in this line here the binary gets downladed, but the releases do not contain plain binaries anymore, but gz files...